src/Pages/Contact/Form.jsx

```diff
@@ -24,7 +24,7 @@ const Form = ({ getSubmissions }) => {
     };
 
     const onSubmit = async (e) => {
-        e.preventDefult();
+        e.preventDefault();
```

Comment:
This was causing the form to not be processed properly and therefore would not successfully add to database